LocalSettings.php

```diff
@@ -4389,6 +4389,23 @@
 	'wgDefaultMobileSkin' => [
 		'default' => 'minerva',
 	],
+	'wgMFNamespacesWithoutCollapsibleSections' => [
+		// See https://github.com/wikimedia/mediawiki-extensions-MobileFrontend?tab=readme-ov-file#wgmfnamespaceswithoutcollapsiblesections
+		'default' => [
+			NS_FILE,
+			NS_CATEGORY,
+			NS_SPECIAL,
+			NS_MEDIA,
+		],
+		'twistwoodtaleswiki' => [
```

Comment:
Can you remove the override for your wiki so I can approve both this and #6113 ?